python/tvm/autotvm/graph_tuner/utils/utils.py

```diff
@@ -77,6 +77,22 @@ def is_boundary_node(node_entry, input_names):
     return out
 
 
+def is_skipped_node(node_entry):
```

Comment:
is this the only entry to access SKIPPED_OP? if so I suggest to directly define SKIPPED_OP in this function. so as LAYOUT_FIXED_OP